### ISSUE TITLE
Issue with default parameter arguments

### ISSUE THREAD

**Issue by bbaliguet** [schmod/babel-plugin-angularjs-annotate#30]
Hi,
There seems to be something wrong with default parameter arguments:

Input code: 
```.js
const outside = function (arg = {}){
  /*@ngInject*/
  const inside = function ($q) {};
  return inside;
}
```

Output code:
```.js
"use strict";

var outside = function outside() {
  var arg = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : {};

  /*@ngInject*/
  var inside = function inside($q) {};
  return inside;
};
```

Removing the default argument does work, `inside.$inject = ["$q"];` appears again.

**Comment by schmod** [comment]
This looks like an upstream (and weird!) bug in Babel (linked above).  I have a workaround and will make a new release in a sec.

**Comment by schmod** [comment]
Fixed in 0.8.1

**Comment by bbaliguet** [comment]
Thanks for the quick fix 😄  !
